clients/src/main/resources/common/message/GetTelemetrySubscriptionRequest.json

```diff
@@ -0,0 +1,31 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 10116,
+  "type": "request",
+  "listeners": ["zkBroker", "broker"],
+  "name": "GetTelemetrySubscriptionRequest",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    {
+      "name": "ClientInstanceId",
+      "type": "uuid",
+      "versions": "0+",
+      "about": "Unique id for this client instance, Must be set to 0 on the very first request"
```

Comment:
```suggestion
      "about": "Unique id for this client instance, must be set to 0 on the very first request."
```

Reply:
Is the uuid protocol type nullable? If so, can we use that instead?
Reason is it is not clear what "set to 0" means here. Should all 128 bits be zeroed?

Reply:
Yes it can be NULL in fact that's what is proposed in the KIP too. My only reasoning to use explicit value either 0 or -1 is to override any serialization/deserialization errors.